### PR TITLE
Mcg fix

### PR DESCRIPTION
**Pull request by ChrisMcGowan**
## Changes proposed in this pull request:

-  Updated aide to 0.17.3
- Added ignore rule syntax
- Changed add rule syntax
- fixed pcre/utf compile issues with bionic
- fixed flock argument issue
- changed default bosh package binary locations
- add syntax to not hardcode aide release ver in packing script

## Security considerations

aide should now work as expected on bionic

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,11 +11,17 @@ This release is made with the following assumptions:
...
-Extra rules can be applied via the `aide_rules` property, which should be just raw text rules:
+Extra rules can be applied via the `add_aide_rules` property, which should be just raw text rules:
 
 ```yaml
-aide_rules: |
+add_aide_rules: |
   /path/to/tacos R # don't mess with my tacos
+
+```
+Extra ignore rules can be applied via the `ignore_aide_rules` property, which should be just raw text rules - if your ignoring a sub file/directory either in a default rule or under an above `add_aide_rule` place your ignore syntax here:
```
Comment (ccostino) [inline README.md:21]: Small grammar fix!

```suggestion
Extra ignore rules can be applied via the `ignore_aide_rules` property, which should be just raw text rules - if you're ignoring a sub file/directory either in a default rule or under an above `add_aide_rule` place your ignore syntax here:
```

Comment (bengerman13) [inline README.md:21]:
```suggestion
Extra ignore rules can be applied via the `ignore_aide_rules` property, which should be just raw text rules - if you're ignoring a sub file/directory either in a default rule or under an above `add_aide_rule` place your ignore syntax here:
```

File: jobs/aide/spec [hunk 2]
```diff
@@ -13,6 +13,9 @@ packages:
 - aide
 
 properties:
-  aide_rules:
-    description: Additional rules for AIDE to apply
+  add_aide_rules:
```
Comment (bengerman13) [inline jobs/aide/spec:16]: I wonder if something like `prepend_rules` and `append_rules` would make more sense?

File: manifest.yml [hunk 3]
```diff
@@ -24,11 +24,12 @@ instance_groups:
   - name: aide
     release: aide
     properties:
-      aide_rules: |
+      ignore_aide_rules: |
+        !/var/vcap/data/stuff
```
Comment (ccostino) [inline manifest.yml:28]: I just want to make sure that the `!` is needed here - I take it that's to indicate to ignore?

**Review by ccostino — COMMENTED** [review]
Thanks, @ChrisMcGowan!  Just a couple of quick things before approving.

File: manifest.yml (hunk 3 above)
Comment (ChrisMcGowan) [reply manifest.yml:28]: yes @ccostino the `!` is to ignore

File: jobs/aide/spec (hunk 2 above)
Comment (ChrisMcGowan) [reply jobs/aide/spec:16]: @bengerman13 that's doable and an easy change - give me a few

**Review by ccostino — DISMISSED** [review]
Thanks, @ChrisMcGowan!

**Pull request merged**